diff --git a/node_modules/@wordpress/env/lib/init-config.js b/node_modules/@wordpress/env/lib/init-config.js
index 9c6885d..543e171 100644
--- a/node_modules/@wordpress/env/lib/init-config.js
+++ b/node_modules/@wordpress/env/lib/init-config.js
@@ -268,6 +268,7 @@ RUN rm /tmp/composer-setup.php`;
 	dockerFileContent += `
 USER $HOST_UID:$HOST_GID
 ENV PATH="\${PATH}:/home/$HOST_USERNAME/.composer/vendor/bin"
+ENV COMPOSER_HOME="/home/$HOST_USERNAME/.composer"
 RUN composer global require --dev phpunit/phpunit:"^5.7.21 || ^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0"
 USER root`;
 
